src/ripple/protocol/SField.h

```diff
@@ -476,12 +484,24 @@ extern SF_AMOUNT const sfHighLimit;
 extern SF_AMOUNT const sfFee;
 extern SF_AMOUNT const sfSendMax;
 extern SF_AMOUNT const sfDeliverMin;
+extern SF_AMOUNT const sfAsset1;
```

Comment:
These aren't great names. Names are hard, I know. But let's take some time to see if we can think of some better names.